lambdas/issuedcredentials/src/main/java/uk/gov/di/ipv/core/issuedcredentials/IssuedCredentialsHandler.java

```diff
@@ -0,0 +1,68 @@
+package uk.gov.di.ipv.core.issuedcredentials;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import com.amazonaws.services.lambda.runtime.RequestHandler;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import uk.gov.di.ipv.core.library.annotations.ExcludeFromGeneratedCoverageReport;
+import uk.gov.di.ipv.core.library.helpers.ApiGatewayResponseGenerator;
+import uk.gov.di.ipv.core.library.service.UserIdentityService;
+
+import java.util.Collections;
+import java.util.Map;
+import java.util.logging.Logger;
+
+import static java.util.logging.Level.WARNING;
+
+public class IssuedCredentialsHandler
+        implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
+
+    private static final Logger LOGGER = Logger.getLogger(IssuedCredentialsHandler.class.getName());
+    private static final Integer OK = 200;
+    private static final Integer BAD_REQUEST = 400;
+
+    public static final String IPV_SESSION_ID_HEADER_KEY = "ipv-session-id";
+
+    private final UserIdentityService userIdentityService;
+
+    public IssuedCredentialsHandler(UserIdentityService userIdentityService) {
+        this.userIdentityService = userIdentityService;
+    }
+
+    @ExcludeFromGeneratedCoverageReport
+    public IssuedCredentialsHandler() {
+        this.userIdentityService = new UserIdentityService();
+    }
+
+    @Override
+    public APIGatewayProxyResponseEvent handleRequest(
+            APIGatewayProxyRequestEvent input, Context context) {
+        Map<String, String> headers = input.getHeaders() == null ? Map.of() : input.getHeaders();
```

Comment:
We have a RequestHelper.getHeaderByKey method that we can use to get the ipv-session-id header value. It makes sure we ignore the case of the header name when trying to read it.

Reply:
👍🏼